test/src/test/java/org/apache/accumulo/harness/SharedMiniClusterIT.java

```diff
@@ -48,6 +50,7 @@
  * a method annotated with the {@link org.junit.BeforeClass} JUnit annotation and {@link #stopMiniCluster()} in a method annotated with the
  * {@link org.junit.AfterClass} JUnit annotation.
  */
+@Category(MiniClusterOnlyTest.class)
```

Comment:
See the java.lang.annotation.Inherited annotation on Category. This is why I also had to bump to junit-4.12 (from 4.11).